README.md

```diff
@@ -1,169 +1,30 @@
-# XMTP-JS remote content types
+# XMTP JS Content Types
 
-This repo provides JavaScript implementations of content types you can use with your app built with the [XMTP client SDK for JavaScript](https://github.com/xmtp/xmtp-js) (`xmtp-js`).
+This repo provides JavaScript implementations of content types you can use with your app built with the XMTP client SDK for [JavaScript](https://github.com/xmtp/xmtp-js) or [React](https://github.com/xmtp/xmtp-web/packages/react-sdk).
 
-Currently, the repo provides a remote attachment content type and will grow to provide more content types.
+To learn more about the contents of this repository, see this README and the READMEs provided for [packages](https://github.com/xmtp/xmtp-js-content-types/tree/main/packages).
 
-## Send a remote attachment
+## What's inside?
 
-Use the [remote-attachment](https://github.com/xmtp/xmtp-js-content-types/tree/main/remote-attachment) package to enable your app to send and receive message attachments.
+### Packages
 
-### What’s an attachment?
+- [`content-type-remote-attachment`](packages/content-type-remote-attachment): This package provides a remote attachment content type.
 
-Attachments are files. More specifically, attachments are objects that have:
+## Requirements
 
-- `filename` Most files have names, at least the most common file types.
-- `mimeType` What kind of file is it? You can often assume this from the file extension, but it's nice to have a specific field for it. [Here's a list of common mime types.](https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types)
-- `data` What is this file's data? Most files have data. If the file doesn't have data then it's probably not the most interesting thing to send.
+- Node 18+
+- Yarn v3+ is required only when working with this repo. See [Yarn Installation](https://yarnpkg.com/getting-started/install), or run the `setup-yarn` script in the `/dev` directory.
 
-### Why remote attachments?
+## Running tests
 
-Because XMTP messages can only be up to 1MB in size, we need to store the attachment somewhere other than the XMTP network. In other words, we need to store it in a remote location.
+Before running unit tests, a required Docker container must be started. To do so, run `yarn test:setup`. Run `yarn test:teardown` to stop the Docker container.
```

Comment:
this might also be obvious to devs - but might this edit be helpful?

```suggestion
Before running unit tests, a required Docker container must be started. To do so, start Docker and run `yarn test:setup`. Run `yarn test:teardown` to stop the Docker container.
```